src/features/game/events/landExpansion/fertiliseFruit.ts

```diff
@@ -0,0 +1,100 @@
+import Decimal from "decimal.js-light";
+import cloneDeep from "lodash.clonedeep";
+import { isFruitReadyToHarvest } from "./fruitHarvested";
+import {
+  FRUIT_COMPOST,
+  FruitCompostName,
+} from "features/game/types/composters";
+import { GameState, PlantedFruit } from "features/game/types/game";
+import { FRUIT } from "features/game/types/fruits";
+
+export enum FERTILISE_FRUIT_ERRORS {
+  EMPTY_PATCH = "Fruit Patch does not exist!",
+  EMPTY_FRUIT = "There is no fruit planted!",
+  READY_TO_HARVEST = "Fruit is ready to harvest!",
+  FRUIT_ALREADY_FERTILISED = "Fruit is already fertilised!",
+  NO_FERTILISER_SELECTED = "No fertiliser selected!",
+  NOT_A_FERTILISER = "Not a fertiliser!",
+  NOT_ENOUGH_FERTILISER = "Not enough fertiliser!",
+}
+
+export type FertiliseFruitAction = {
+  type: "fruit.fertilised";
+  patchID: string;
+  fertiliser: FruitCompostName;
+};
+
+type Options = {
+  state: Readonly<GameState>;
+  action: FertiliseFruitAction;
+  createdAt?: number;
+};
+
+const getYield = (fruitDetails: PlantedFruit, fertiliser: FruitCompostName) => {
+  if (fertiliser === "Fruitful Blend") {
+    return fruitDetails.amount + 0.25;
```

Comment:
Double-check to make sure this is not introducing "float drama".  Should it be `Decimal("0.25")`?